taipy/core/exceptions/exceptions.py

```diff
@@ -186,7 +186,7 @@ def __init__(self, sequence_id: str):
 class NonExistingSequence(Exception):
     """Raised if a requested Sequence is not known by the Sequence Manager."""
 
-    def __init__(self, sequence_id: str, scenario_id: str=None):
+    def __init__(self, sequence_id: str, scenario_id: str = None):
```

Comment:
That's not the convention. In the default values definition, in method signatures, there is no space around the '='.

Reply:
oh? not sure why this was changed!